### DISCUSSION TITLE
[이길어때] SSE 방식의 실시간 알림 구현하기

### DISCUSSION THREAD

**Discussion by giscus** [stoneHee99/tistory_comment#5] [Comments]
# [이길어때] SSE 방식의 실시간 알림 구현하기

이길어때가 더 궁금하다면? [이길어때] 목차 프로젝트 이길어때의 개발 과정을 정리합니다. 서비스 주소 📍 https://yigil.co.kr/ 이길로그 나의 너무 멋진 장소 yigil.co.kr 깃허브 주소 🐈‍⬛ https://github.com/Kernel360/f1-Yigil GitHub - Kernel360/f1-Yigil Con stonehee99.tistory.com 제가 진행하고 있는 이길어때 프로젝트는 점차 기능을 갖춰가면서 하나의 SNS 서비스의 형태로 발전해나가고 있답니다 :) SNS하면 또 실시간으로 사용자들의 반응을 확인하고 싶은 요구가 생기기 마련인데요, 그에 맞추어 저희 서비스에서도 실시간 알림 기능을 기획하였답니다. 오늘은 프로젝트를 진행하면서 SSE를 통한 알림 기능을 구현한..

https://stonehee99.tistory.com/30?utm_source=perplexity

**Comment by gyungsubLee** [comment]
토이 프로젝트로 알람 기능에 대해서 구현을 맞게 되었는데, 해당 글에서 많은 도움을 받았네요  좋은 글 감사합니다

**Reply by stoneHee99** [reply]
도움이 많이 되었다니 다행이네요 :)
